tests/integration/old/fake_services/permission.py

```diff
@@ -8,8 +10,13 @@ class PermissionTestAdapter:
     implementation.
     """
 
-    def __init__(self, superuser: int, *args: Any, **kwargs: Any) -> None:
-        self.superuser = superuser
+    def __init__(self, superuser_id: int, *args: Any, **kwargs: Any) -> None:
+        self.superuser_id = superuser_id
 
-    def check_action(self, user_id: int, action: str, data: Any) -> bool:
-        return user_id == self.superuser
+    def is_allowed(
+        self, name: str, user_id: int, data: List[Dict[str, Any]]
+    ) -> List[Dict[str, Any]]:
+        if user_id == self.superuser_id:
+            return [{} for _ in data]
+        else:
+            raise NotAllowed("Not the superuser")
```

Comment:
This is the fake adapter from the old integration tests. The changes here will have no effect on the actual tests

Reply:
Oh ok, but I'll leave them, if they do not hurt (It is more work to redo them instead of leaving as-is;) )

Reply:
Yes, just wanted to let you know :D